tests/js/spec/views/performance/landing/index.spec.tsx

```diff
@@ -153,6 +155,21 @@ describe('Performance > Landing > Index', function () {
     expect(wrapper.find('Table').exists()).toBe(true);
   });
 
+  it('renders react-native table headers in mobile view', async function () {
```

Comment:
Not sure about the value of this test. I'd love to be able to mock up a view knowing it's supposed to display a react-native view and then pull the table headers from the DOM. My issue with this is that it's only testing implementation details. I'd appreciate suggestions on how to make this better.